run_dir/static/js/running_notes.js

```diff
@@ -117,7 +118,7 @@ function load_running_notes(wait) {
 function preview_running_notes(){
     var now = new Date();
     $('.todays_date').text(now.toDateString() + ', ' + now.toLocaleTimeString());
-    var category = generate_category_label($('#rn_category option:selected').val());
+    var category = generate_category_label($('.btnCatFilter.active').val());
```

Comment:
* Update CSS selector for new class name.
* Use `.text()` instead of `.val()` since we deleted the `value=""` attributes.

```suggestion
    var category = generate_category_label($('.rn-cat-filter .active').text());
```